iac-extensions/terraform/src/test/resources/checks/UnversionedS3BucketCheck/test_v4.tf

```diff
@@ -0,0 +1,38 @@
+resource "aws_s3_bucket" "mynoncompliantbucket" {
```

Comment:
I'd like to have minimal changes between `test_v3.tf` and `test_v4.tf`.
Here the name of the resource is the same for the 2 files even if the resource is compliant.
I think we can find better names for resources, here for an example: `bucket_without_any_versioning`.